pkg/analytics/seed_test.go

```diff
@@ -36,7 +36,7 @@ func createMemberlist(t *testing.T, port, memberID int) *memberlist.KV {
 	var cfg memberlist.KVConfig
 	flagext.DefaultValues(&cfg)
 	cfg.TCPTransport = memberlist.TCPTransportConfig{
-		BindAddrs: []string{"localhost"},
+		BindAddrs: []string{"0.0.0.0"},
```

Comment:
What's the reason for this change?

Reply:
See https://github.com/grafana/dskit/pull/396. It's a security fix.

Reply:
Huh? The security fix was to correctly bind to localhost, precisely to _not_ bind on all interfaces inadvertently. In any case, this is a test so no worries, was just curious.